docs/tutorials/security-with-pro.rst

```diff
@@ -0,0 +1,611 @@
+
+Use Multipass to create a VM
+============================
+
+We're going to create our VM using the Ubuntu 18.04 LTS (Bionic Beaver). It is
+the most recent LTS to reach the end of its standard support period (as of the
+time of writing), so it has updates available through the Ubuntu Pro Expanded
+Security Maintenance (ESM) services. 
+
+Let's start by creating our VM by running the following command:
+
+.. code-block:: bash
+
+   $ multipass launch bionic --name test-vm
+
+This command will create (``launch``) a Bionic VM that we will label with the
+name "``test-vm``".
+
+It will probably take a few minutes to fetch the Bionic image (depending on
+your internet speed). Once Multipass has successfully set up the VM, you will
+see a message like this in your terminal:
+
+.. code-block:: text
+
+   Launched: test-vm
+
+We can now enter our new VM with this command:
+
+.. code-block:: bash
+
+   $ multipass shell test-vm
+
+Notice that when we run this command, our terminal username and hostname
+change to ``ubuntu@test-vm``, which tells us we are now operating inside the
+VM. 
+
+We then see a welcome message that looks similar to this (throughout this
+tutorial, we will truncate the output for brevity -- removed sections will be
+indicated with ``[...]``):
+
+.. code-block:: text
+
+    Welcome to Ubuntu 18.04.6 LTS (GNU/Linux 4.15.0-212-generic x86_64)
+
+     [...]
+
+      System information as of Wed Jun 19 14:17:26 BST 2024
+
+      [...]
+
+    Expanded Security Maintenance for Infrastructure is not enabled.
+
+    16 updates can be applied immediately.
+    11 of these updates are standard security updates.
+    To see these additional updates run: apt list --upgradable
+
+    109 additional security updates can be applied with ESM Infra.
+    Learn more about enabling ESM Infra service for Ubuntu 18.04 at
+    https://ubuntu.com/18-04
+
+    [...]
+
+We can see immediately that our virtual machine is not fully up-to-date, with
+16 software updates that can be applied, and 103 additional security updates
+(available with ESM-Infra). Let's first see which software updates we can
+apply by running the command that was suggested in the output:
+
+.. code-block:: bash
+
+   $ apt list --upgradable
+
+This will show us a list of all the packages with available updates, the
+current version, and the version we can upgrade to:
+
+.. code-block:: bash
+
+    Listing... Done
+    [...]
+    python3-update-manager/bionic-updates 1:18.04.12 all [upgradable from: 1:18.04.11.13]
+    shim-signed/bionic-updates 1.37~18.04.13+15.7-0ubuntu1 amd64 [upgradable from: 1.37~18.04.11+15.4-0ubuntu9]
+    ubuntu-advantage-tools/bionic-updates 32.3~18.04 amd64 [upgradable from: 27.14.4~18.04]
+    update-manager-core/bionic-updates 1:18.04.12 all [upgradable from: 1:18.04.11.13]
+    update-notifier-common/bionic-updates 3.192.1.21 all [upgradable from: 3.192.1.19]
+
+One of the packages that needs updating is the ``ubuntu-advantage-tools``
```

Comment:
@s-makin maybe we should starting to advertise the `ubuntu-pro-client` package here instead of the old one.

what do you think @orndorffgrant @renanrodrigo @dheyay ?

Reply:
That's a great point, and yes - definitely we should!